lib/chat/message.dart

```diff
@@ -16,50 +17,54 @@ class MessageView extends StatelessWidget {
   Widget build(BuildContext context) {
     return Align(
       alignment: isSent ? Alignment.centerRight : Alignment.centerLeft,
-      child: type == textMessage ? Container(
-        decoration: BoxDecoration(
-          color: isSent ? Theme.of(context).accentColor : Colors.grey,
-          borderRadius: BorderRadius.all(Radius.circular(8.0)),
-        ),
-        margin: isSent 
-          ? EdgeInsets.fromLTRB(50.0, 10.0, 10.0, 10.0) 
-          : EdgeInsets.fromLTRB(10.0, 10.0, 50.0, 10.0),
-        padding: EdgeInsets.all(10.0),
-        child: Text(
-          message,
-          style: GoogleFonts.muli(
-            textStyle: TextStyle(
-              color: Colors.white, 
-              letterSpacing: .5, 
-              fontSize: 14.0, 
+      child: type == textMessage ? 
+        Bubble(
+          nip: isSent ? BubbleNip.rightTop : BubbleNip.leftTop,
+          nipOffset: 0.0,
+          style: BubbleStyle(
+            margin: isSent ? 
+              BubbleEdges.fromLTRB(50.0, 10.0, 10.0, 10.0) :
+              BubbleEdges.fromLTRB(10.0, 10.0, 50.0, 10.0),
+            padding: BubbleEdges.all(10.0),
+            color: isSent ? Theme.of(context).accentColor : Colors.grey,
+          ),
+          child: Text(
+            message,
+            style: GoogleFonts.muli(
+              textStyle: TextStyle(
+                color: Colors.black, 
+                letterSpacing: .5, 
+                fontSize: 14.0, 
+              )
             )
           )
-        )
-      ) : Container(
-        decoration: BoxDecoration(
-          color: isSent ? Theme.of(context).accentColor : Colors.grey,
-          borderRadius: BorderRadius.all(Radius.circular(8.0)),
-        ),
-        margin: isSent ? 
-          EdgeInsets.fromLTRB(50.0, 10.0, 10.0, 10.0) :
-          EdgeInsets.fromLTRB(10.0, 10.0, 50.0, 10.0),
-        child: ClipRRect(
-          borderRadius: BorderRadius.all(Radius.circular(8.0)),
-          child: CachedNetworkImage(
-            imageUrl: message,
-            fit: BoxFit.fill,
-            placeholder: (context, url) => Container(
-              width: 200.0,
-              height: 200.0,
-              padding: EdgeInsets.all(70.0),
-              child: CircularProgressIndicator(
-                valueColor: AlwaysStoppedAnimation<Color>(Colors.white)
-              )
-            ),
-            errorWidget: (context, url, error) => Text("error"),
+        ) : Bubble(
+          nip: isSent ? BubbleNip.rightTop : BubbleNip.leftTop,
+          nipOffset: 0.0,
+          style: BubbleStyle(
+            margin: isSent ? 
```

Comment:
Since `isSent` is being used multiple times in this file, this can be checked at the beginning of the widget to determine which styling to use.